_docker/docker-compose.yml.erb

```diff
@@ -29,9 +29,9 @@ awestruct:
     - github_token
     - drupal_user
     - drupal_password
-    - AWESTRUCT_HOST_PORT
-    - SEARCHISKO_HOST_PORT
-    - DRUPAL_HOST_PORT
+    - AWESTRUCT_HOST_PORT=<%=ENV['AWESTRUCT_HOST_PORT']%>
+    - SEARCHISKO_HOST_PORT=<%=ENV['SEARCHISKO_HOST_PORT']%>
+    - DRUPAL_HOST_PORT=<%=ENV['DRUPAL_HOST_PORT']%>
```

Comment:
Got it. No problem

On Friday, October 2, 2015, Oliver Shaw <notifications@github.com> wrote:

> In _docker/docker-compose.yml.erb
> <https://github.com/redhat-developer/developers.redhat.com/pull/448#discussion_r41039651>
> :
>
> > @@ -29,9 +29,9 @@ awestruct:
> >      - github_token
> >      - drupal_user
> >      - drupal_password
> > -    - AWESTRUCT_HOST_PORT
> > -    - SEARCHISKO_HOST_PORT
> > -    - DRUPAL_HOST_PORT
> > +    - AWESTRUCT_HOST_PORT=<%=ENV['AWESTRUCT_HOST_PORT']%>
> > +    - SEARCHISKO_HOST_PORT=<%=ENV['SEARCHISKO_HOST_PORT']%>
> > +    - DRUPAL_HOST_PORT=<%=ENV['DRUPAL_HOST_PORT']%>
>
> Thanks for the comments so far.
>
> For this particular one, the reason that these environments are set is to
> allow the ports to remain consistent if someone is restarting awestruct.
> All the other env variables are ok to set again in the environment. The
> ports would be different as the services will already be running.
>
> I've probably not explained it too well, but we need those ports set :-)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/redhat-developer/developers.redhat.com/pull/448/files#r41039651>
> .
>


-- 
Sent from Gmail Mobile